RELEASES.md

```diff
@@ -0,0 +1,131 @@
+# 🚀 Releases
+
+This document outlines the Moby project’s release process, including cadence and versioning.
+
+## 📌 Versioning
+
+Releases of moby modules will be versioned using dotted triples, similar to
+[Semantic Version](http://semver.org/). For the purposes of this document, we
+will refer to the respective components of this triple as
+`<major>.<minor>.<patch>-[suffix[N]]`. The version number may have additional
+information, such as alpha, beta and release candidate qualifications along with
+a numeric identifier. Such releases will be considered "pre-releases".
+
+> [!NOTE]
+> v17.03-v20.10 releases were following [CalVer](https://calver.org/).
+> v2.0.0 introduces the `github.com/moby/moby/v2` go module after the
+> previous v28 release of the `github.com/docker/docker` pseudo-module.
+
+## ⏳ Release cadence
+
+The project will release as quickly as possible while ensuring stability and
+timely feature delivery. Maintainers curate a feature roadmap and set milestones
+for each release.
+
+Release dates assigned to milestones are commitments, and any changes will be
+announced to the community via an
+[announcement discussion](https://github.com/moby/moby/discussions/categories/announcements).
+To meet deadlines, some features may be deferred to future releases.
+
+If your issue or feature is missing from the roadmap, please open a GitHub issue
+or comment to request its inclusion.
+
+## 🎯 Major and minor releases
+
+Major and minor releases of moby will be made from main. Releases will be marked
+with GPG signed tags and posted at https://github.com/moby/moby/releases. The
+tag will be of the format `v<major>.<minor>.<patch>-[suffix[N]]`.
+
+After a minor release of the main module, a branch will be created, with the format
+`release/<major>.<minor>` from the minor tag. All further patch releases will be
+done from that branch. For example, after the release of `v2.1.0`, a branch
+`release/2.1` would be created from that tag. All future patch releases will be
```

Comment:
This part (versioning) we still need to look closely at.

Given that we provided NO Go API stability, we can decide to stay on "v2 forever".

At least we must avoid reaching v17, because then we'll collide with v17.12.0, which ISTR go modules considered SemVer (but +incompatible)

Reply:
Or maybe we just can't count... v15, v16, v29, v30 😆 

I'm hoping being clear about what we consider "supported" helps us define this. We respect can both semver and  take compatibility seriously by communicating what exactly the contract is and recommending against importing unsupported packages. I think before we hit a v3 we could probably just stabilize the Go interface with proper use of internal packages, or even call it v3 at that point.

Reply:
It'll take some time for the Go interface to shake out. Users will provide feedback on what they like and dislike and we will need some flexibility to change it. In the meantime, the client and API are stable and satisfy a majority of users.

> Or maybe we just can't count... v15, v16, v29, v30 😆

It's unlikely I'll see v16 + 1 in my lifetime at least skipping is a simple enough solution. Or maybe Go will have swapped to epoch semver by then.